models/models.go

```diff
@@ -1,6 +1,62 @@
 package models
 
+import (
+	"time"
+)
+
 type Location struct {
-	Latitude string 	`json:"latitude"`
-	Longitude string	`json:"longitude"`
+	Latitude  string `json:"latitude"`
+	Longitude string `json:"longitude"`
+}
+
+type DBModel struct {
+	ID        uint       `gorm:"primary_key"`
+	CreatedAt time.Time  `json:"-"`
+	UpdatedAt time.Time  `json:"-"`
+	DeletedAt *time.Time `json:"-"`
+}
+
+type Rider struct {
+	DBModel
+	Name        string
+	Source      string
+	Destination string
+	//InTaxiNum   int // In which taxi user is riding in
+	MaximumWaitingTime int // Time in minutes, later we can try time
```

Comment:
In the future, need following fields "gender", "age", "nationality", "smoke need", "number of times violating policies", "other requests"